server/services/comment-service.ts

```diff
@@ -21,4 +22,32 @@ export abstract class CommentService {
       )
       .catch((reason) => logger('Rejected comments:', reason));
   }
+
+  static async postComment(
+    comment: CommentModel,
+    submitterId: string,
+    context: IGraphQlSchemaContext
+  ) {
+    const id = Math.floor(Math.random() * 10000);
```

Comment:
I'm trying to make the random id in here for the local comment. However, this generate Id can be improved more